### DISCUSSION TITLE
GH Discussions over Gitter

### DISCUSSION THREAD

**Discussion by dmitry-timofeev** [jni-rs/jni-rs#297] [Ideas]
@ndarilek comment [here](https://github.com/jni-rs/jni-rs/issues/257#issuecomment-752643374) highlighted an issue with Gitter: lack of searchable history. 

@stanislav-tkach Shall we replace Gitter ref with GH Discussions? Watch settings allow to subscribe to that:
<img width="443" alt="изображение" src="https://user-images.githubusercontent.com/7531824/105399904-e7128880-5c2c-11eb-98b4-e9488530019f.png">

**Comment by stanislav-tkach** [comment]
I haven't really tried GitHub Discussions yet, but I'm surprised that people are still using jni-rs Gitter. 🙃 I suppose it makes sense to pin the message with a link to the discussions page.

**Reply by dmitry-timofeev** [reply]
I guess the reason is that we have a clickable banner pointing to the room in our readme 🙃 Gotta fix that then :)

**Comment by ndarilek** [comment]
Note that Gitter is now Matrix-based, so I wouldn't be surprised if more 
folks started using it. I myself now regularly use a few Gitter rooms 
since I can join from Element. That said, maybe it isn't a good fit for you.

**Comment by rib** [comment]
I've been lurking in the Gitter room for a while now, since just before the 0.20 release, and in general it looks like there's very little Gitter activity. Personally I also find it awkward to monitor while I don't use Gitter for anything else.

I've announced the last few releases there and in that time just answered one very basic question.

After announcing the latest 0.21.1 patch release today I decided to also update the topic for the Gitter room to sign post people towards these Discussion boards instead.

Overall I think these Discussion boards are probably a much better place for people to open discussions that don't belong in issues or pull requests since I think they will have broader visibility amongst the different contributors to this project and discussions can have a much clearer resolution that others will be able to follow.
